msg/templates/uorb_microcdr/microRTPS_client.cpp.em

```diff
@@ -166,6 +196,49 @@ void *send(void *args)
 		}
 @[    end for]@
 
+@[    if send_topics_poll]@
+		{
+			px4_pollfd_struct_t fds[@(len(send_topics_poll))];
+@[        for idx, topic in enumerate(send_topics_poll)]@
+			fds[@(idx)].fd = subs_poll->@(topic)_sub;
+			fds[@(idx)].events = POLLIN;
+@[        end for]@
+
+			int pret = px4_poll(fds, @(len(send_topics_poll)), 20);
+
+			if (pret < 0) {
+				PX4_ERR("poll failed (%i)", pret);
```

Comment:
There should be a sleep here, to avoid busy-looping

Reply:
Done. To not hardcode a value in the sleep period I've used `tx_interval`.